src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_json_debug_options.py

```diff
@@ -0,0 +1,132 @@
+import sys
+import platform
+try:
+    import urllib
+    urllib.unquote  # noqa
+except Exception:
+    import urllib.parse as urllib
+
+
+def bool_parser(s):
+    return s in ("True", "true", "1")
+
+
+if sys.version_info >= (3,):
+
+    def unquote(s):
+        return None if s is None else urllib.unquote(s)
+
+else:
+
+    # In Python 2, urllib.unquote doesn't handle Unicode strings correctly,
+    # so we need to convert to ASCII first, unquote, and then decode.
+    def unquote(s):
+        if s is None:
+            return None
+        if not isinstance(s, bytes):
+            s = bytes(s)
+        s = urllib.unquote(s)
+        if isinstance(s, bytes):
+            s = s.decode('utf-8')
+        return s
+
+DEBUG_OPTIONS_PARSER = {
+    'WAIT_ON_ABNORMAL_EXIT': bool_parser,
+    'WAIT_ON_NORMAL_EXIT': bool_parser,
+    'REDIRECT_OUTPUT': bool_parser,
+    'VERSION': unquote,
+    'INTERPRETER_OPTIONS': unquote,
+    'WEB_BROWSER_URL': unquote,
+    'DJANGO_DEBUG': bool_parser,
+    'FLASK_DEBUG': bool_parser,
+    'FIX_FILE_PATH_CASE': bool_parser,
+    'CLIENT_OS_TYPE': unquote,
+    'DEBUG_STDLIB': bool_parser,
+    'STOP_ON_ENTRY': bool_parser,
+    'SHOW_RETURN_VALUE': bool_parser,
+    'MULTIPROCESS': bool_parser,
+}
+
+DEBUG_OPTIONS_BY_FLAG = {
+    'RedirectOutput': 'REDIRECT_OUTPUT=True',
+    'WaitOnNormalExit': 'WAIT_ON_NORMAL_EXIT=True',
+    'WaitOnAbnormalExit': 'WAIT_ON_ABNORMAL_EXIT=True',
+    'Django': 'DJANGO_DEBUG=True',
+    'Flask': 'FLASK_DEBUG=True',
+    'Jinja': 'FLASK_DEBUG=True',
+    'FixFilePathCase': 'FIX_FILE_PATH_CASE=True',
+    'DebugStdLib': 'DEBUG_STDLIB=True',
+    'WindowsClient': 'CLIENT_OS_TYPE=WINDOWS',
+    'UnixClient': 'CLIENT_OS_TYPE=UNIX',
+    'StopOnEntry': 'STOP_ON_ENTRY=True',
+    'ShowReturnValue': 'SHOW_RETURN_VALUE=True',
+    'Multiprocess': 'MULTIPROCESS=True',
+}
```

Comment:
@karthiknadig I agree with the change, but if possible I'd like to leave the API unchanged for this refactoring and create a separate ticket to address this (this is already a pretty big change).

Do you think that's Ok?

Reply:
That is Ok. Let make sure we have a work item to remove this part of the code. Note: this will have to be done across VS, VSC, and debugger at the same time.